ontology/dns.tf

```diff
@@ -0,0 +1,21 @@
+data "google_dns_managed_zone" "dns_zone" {
+  count    = var.enable && ! var.preview ? 1 : 0
```

Comment:
You can leave out all the preview stuff for now, it will likely change before we incorporate ontology into the preview environments.